api/src/controllers/user.js

```diff
@@ -44,6 +45,73 @@ function logoutCookieOptions() {
   }
 }
 
+function updateUserDebugInfos(req, user) {
+  if (req.headers.platform === "android") {
+    try {
+      z.optional(z.string()).parse(req.body.apilevel);
```

Comment:
```suggestion
      z.optional(z.number()).parse(req.body.apilevel);
```

D'après mes tests apinumber est un nombre (et donc ça plantait)